File: SoulCore/src/main/java/de/soulsmp/soulcore/SoulCore.java
package de.soulsmp.soulcore;

import net.fabricmc.api.ModInitializer;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class SoulCore implements ModInitializer {

    public static final Logger LOGGER = LoggerFactory.getLogger("SoulCore");

    @Override
    public void onInitialize() {
        LOGGER.info("SoulCore initialized!");
    }
}

File: SoulCore/src/main/kotlin/de/soulsmp/soulcore/SoulCore.kt
package de.soulsmp.soulcore

import net.fabricmc.api.ModInitializer
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class SoulCore : ModInitializer {
    override fun onInitialize() {
        LOGGER.info("SoulCore initialized!")
    }

    companion object {
        val LOGGER: Logger = LoggerFactory.getLogger("SoulCore")
    }
}
